packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -462,6 +475,87 @@ export default class OverTheBoxDetailsCtrl {
       });
   }
 
+  /**
+   * Get last seen values
+   */
+  getLastSeen() {
+    if (this.device && this.device.lastSeen) {
+      // check lastSeen access is less than 5 minutes
+      const currentDate = moment();
+      const lastSeen = moment(this.device.lastSeen);
+      const diffDate = currentDate.diff(lastSeen, 'minute');
+
+      this.lastSeenAccess = {
+        lastSeen: diffDate,
+        lastSeenHuman: moment.duration(currentDate.diff(lastSeen)).humanize(),
+      };
+
+      if (diffDate < this.OVERTHEBOX_DETAILS.lastSeen.limit) {
+        this.lastSeenAccess.isRecent = true;
+      } else {
+        this.lastSeenAccess.isRecent = false;
+      }
+    }
+  }
+
+  /**
+   * Check public IP to retrieve its status
+   */
+  checkPublicIP() {
+    return this.OvhApiIp.v6()
+      .query({
+        'routedTo.serviceName': this.serviceName,
+        type: 'overthebox',
+      })
+      .$promise.then((ips) => {
+        const [ip] = ips;
+
+        if (!ip) {
+          this.serviceIP = {
+            status: this.OVERTHEBOX_DETAILS.serviceIpStatus.unknown,
+          };
+        } else if (this.device && this.device.publicIp) {
+          if (ip.includes(this.device.publicIp)) {
+            this.serviceIP = {
+              status: this.OVERTHEBOX_DETAILS.serviceIpStatus.locked,
+            };
+          } else {
+            this.serviceIP = {
+              status: this.OVERTHEBOX_DETAILS.serviceIpStatus.warning,
+            };
+          }
+        } else {
+          this.serviceIP = {
+            status: this.OVERTHEBOX_DETAILS.serviceIpStatus.unknown,
+          };
+        }
+        return ips;
+      })
+      .catch((error) => {
+        this.error.checking = error.data;
+        this.TucToast.error(
+          [
+            this.$translate.instant('an_error_occured'),
+            error.data.message,
+          ].join(' '),
```

Comment:
Once again, you should use string interpolation

Reply:
like that 
this.TucToast.error(
  `${this.$translate.instant('an_error_occured')} ${
   error.data.message}`,
);

Reply:
Yes :) (Don't forget backticks `)

Reply:
like that 
this.TucToast.error(
  `${this.$translate.instant('an_error_occured')} ${
   error.data.message}`,
);